schemas/rules/operators/boolean/and.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://eq.ons.gov.uk/rules/operators/boolean/and.json",
+  "description": "Evaluates boolean `and`, takes two or more arguments",
```

Comment:
Would a full stop not make this read better rather than a comma? Does it need the boolean in the description, considering all these conditions are in the boolean folder? Maybe "Evaluates `and` operator. Takes two or more arguments

Reply:
I have removed the bit about number of arguments from all schemas as it is already documented by the schema restrictions - https://github.com/ONSdigital/eq-questionnaire-validator/pull/96/commits/9802ad6280387a8199a7233aa661ca16f8c48ad2